about.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>WD Course</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <img src="images/Icon-Cancel.svg" id="close-menu" alt="">
+            <div class="up-nav">
+                <div class="img-contact">
+                    <img src="images/facebook.svg" alt="">
+                    <img src="images/twitter.svg" alt="">
+                </div>
+                <ul>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+            <div class="down-nav">
+                <div class="logo">
+                    <a href="home.html"><img src="images/logo.png" alt=""></a>
+                    </div>
+                <div class="bar">
+                    <ul>
+                        <li id="about"><a href="about.html">About</a></li> 
+                        <li id="home"><a href="home.html">Program</a></li> 
+                        <li>Join</li>
+                        <li>Sponsor</li>
+                        <li>News</li>
+                    </ul>
+                </div>
+                <a href="#" id="campaign">WD Campaign</a>
+            </div>
+            
+        </nav>
+    </header>
+
+    <section class="s1">
+        <div class="home-about">
+            <img src="images/Icon - Menu.svg" id="open-menu" alt="">
+             <h2>"Hello! Change world"</h2>
+             <p>Web development course using HTML, CSS, and JavaScript</p>
```

Comment:
- [ ]  Kindly make this element an `h1` instead of `p` since this is the main header of the page 👍